lab5/commands/ExecuteScript.java

```diff
@@ -0,0 +1,34 @@
+package org.example.commands;
+
+import org.example.utils.CollectionManager;
+import org.example.utils.CommandManager;
+import org.example.utils.FileManager;
+
+import java.util.ArrayList;
+
+public class ExecuteScript implements Commands{
+    private final CollectionManager collectionManager;
+    private final CommandManager commandManager;
+
+    public ExecuteScript(CollectionManager collectionManager,
+                         CommandManager commandManager) {
+        this.collectionManager = collectionManager;
+        this.commandManager = commandManager;
+    }
+
+    @Override
+    public void execute() {
+
+    }

```

Comment:
Make abstract classes for the Command interface so you don't need to put these empty implementations everywhere